dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/XtxtUML.xtext

```diff
@@ -48,6 +49,17 @@ TUClass :
 	'}' ';'? | ';')
 ;
 
+TUEnumeration :
+	'enum' name=ValidID ('{'
+		(literals+=TUEnumerationLiteral ','?)* 
```

Comment:
It may look better in my opinion to end every line with a comma:

```java
enum E {
  A,
  B,
  C,
}
```

For a similar example, Java allows array initializers to end with an extra comma:

```java
A[] array = new A[] {
  new A(...),
  new A(...),
  new A(...),
};
```